### ISSUE TITLE
Single letter iterator variables should be allowed where curly braces are allowed

### ISSUE THREAD

**Issue by ed-bassett** [vinomofo/guides#1]

**Comment by gerrywastaken** [comment]
@ed-bassett I don't understand the above edit.

**Comment by icaruswings** [comment]
I think the resolution here was that we are OK with abbreviations in cases where curly braces are OK. I guess, when your enumerable etc, can fit on a single line nicely

**Comment by gerrywastaken** [comment]
@icaruswings Yup I remember the offline convo, I'm just not sure what was meant by the crossing out of the suggestion and then adding a link to the style page. I'm also asking for anybody who comes back here in the future, who I imagine would find this very confusing.
